### ISSUE TITLE
source_id registration of CAMS_CSM1-0

### ISSUE THREAD

**Issue by durack1** [WCRP-CMIP/CMIP6_CVs#246]
```
From: lijd@camsxxx.cn"
Date: Tuesday, March 14, 2017 at 12:47 AM
To: cmor
Subject: register for cmip6

Hello,

We would like to register the institution_ids and source_id for CMIP6. The information is as follows:

"CAMS_CSM1-0":{
"label_extended":"CAMS_CSM 1.0",
"atmospheric_chemistry":"None",
"atmosphere":"ECHAM5_CAMS (T106, 320 x 160; 31 levels; top level 10 mb)",
"ocean_biogeochemistry":"none",
"release_year":"2016",
"cohort":[
""
],
"sea_ice":"SIS",
"label":"CAMS_CSM 1.0",
"institution_id":[
"CAMS"
],
"land_surface":"CoLM",
"aerosol":"None",
"source_id":"CAMS_CSM1-0",
"ocean":"MOM4 (tripolar, 360x200, primarily 1deg latitude/longitude, down to 1/3deg within 30deg of the equatorial tropics; 50 levels; top grid cell 0-10 m)",
"land_ice":"None"
}

Kind regards,

Jianduo Li

Dr. Jianduo Li, Assistant Research Scientist
State Key Laboratory of Severe Weather
Chinese Academy of Meteorological Sciences
46 Zhongguancun South Ave., Beijing 100081
Tel: +86 10-6840-90xx
Fax:+86 10-5899-97xx
```

**Comment by durack1** [comment]
Hi Jianduo,

My suggested example from your email above is below (you can see all registered models at [CMIP6_source_id.json](https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_source_id.json)). The specifications outlining each of the entries can be viewed [here](https://github.com/WCRP-CMIP/CMIP6_CVs/issues/new).

We are hoping to gather version component information, so please add details below, replacing the placeholder text `###PLEASE INCLUDE VERSION INFORMATION###`.

```
        "CAMS_CSM1-0":{
            "aerosol":"None",
            "atmosphere":"ECHAM5_CAMS (T106, 320 x 160 longitude/latitude; 31 levels; top level 10 mb)",
            "atmospheric_chemistry":"None",
            "cohort":[
                "Registered"
            ],
            "institution_id":[
                "CAMS"
            ],
            "label":"CAMS_CSM 1.0",
            "label_extended":"CAMS_CSM 1.0",
            "land_ice":"None",
            "land_surface":"CoLM ###PLEASE INCLUDE VERSION INFORMATION###",
            "ocean":"MOM4 (tripolar, 360 x 200 longitude/latitude, primarily 1deg latitude/longitude, down to 1/3deg within 30deg of the equatorial tropics; 50 levels; top grid cell 0-10 m)",
            "ocean_biogeochemistry":"None",
            "release_year":"2016",
            "sea_ice":"SIS ###PLEASE INCLUDE VERSION INFORMATION###",
            "source_id":"CAMS_CSM1-0"
},
```

**Comment by jianduoli** [comment]
Hi,
The model information is 
"land_surface":"CoLM 1.0",
"sea_ice":"SIS 1.0",
and please set the "activity_participation", "nominal_resolution_atmos" and "nominal_resolution_ocean"  to be "".
Is that fine?

Many thanks!

Jianduo

**Comment by durack1** [comment]
@jianduoli great, so we are close. Now we have:
```
        "CAMS_CSM1-0":{
            "activity_participation":[
                "" ### See activity list at https://github.com/WCRP-CMIP/CMIP6_CVs/blob/master/CMIP6_activity_id.json ###
            ],
            "aerosol":"None",
            "atmosphere":"ECHAM5_CAMS (T106, 320 x 160 longitude/latitude; 31 levels; top level 10 mb)",
            "atmospheric_chemistry":"None",
            "cohort":[
                "Registered"
            ],
            "institution_id":[
                "CAMS"
            ],
            "label":"CAMS_CSM 1.0",
            "label_extended":"CAMS_CSM 1.0",
            "land_ice":"None",
            "land_surface":"CoLM 1.0",
            "nominal_resolution_atmos":[
                "" ### See Appendix 2 in https://goo.gl/v1drZl ###
            ],
            "nominal_resolution_landIce":[
                "None"
            ],
            "nominal_resolution_ocean":[
                "" ### See Appendix 2 in https://goo.gl/v1drZl ###
            ],
            "ocean":"MOM4 (tripolar, 360 x 200 longitude/latitude, primarily 1deg latitude/longitude, down to 1/3deg within 30deg of the equatorial tropics; 50 levels; top grid cell 0-10 m)",
            "ocean_biogeochemistry":"None",
            "release_year":"2016",
            "sea_ice":"SIS 1.0",
            "source_id":"CAMS_CSM1-0"
},
```
We require entries for `"activity_participation"`, `"nominal_resolution_atmos" and `"nominal_resolution_ocean"`, so will need these completed before registering your model. To start we can just list `CMIP` for activity participation, assuming you will only be contributing to the CMIP6 DECK simulation stack with this model

**Comment by jianduoli** [comment]
Please set "activity_participation", "nominal_resolution_atmos" and "nominal_resolution_ocean" as:
"activity_participation": ["CMIP"],
"nominal_resolution_atmos":["100km"],
"nominal_resolution_ocean":["100km"],

Many thanks!

Jianduo

**Comment by durack1** [comment]
@jianduoli your model is now registered. Please take one last look at the registered entry by browsing [CMIP6_source_id.html](http://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_source_id.html) and let me know if there are any other tweaks required

**Comment by durack1** [comment]
@jianduoli we noted an inconsistency with the format of the `CAMS_CSM1-0` registration, and have now corrected this across to `CAMS-CSM1-0`. Please check your complete model registration details in [CMIP6_source_id.html](https://rawgit.com/WCRP-CMIP/CMIP6_CVs/master/src/CMIP6_source_id.html) and get back to me with any further tweaks required.

For reference, this issue was caught and resolved in WCRP-CMIP/CMIP6_CVs#466
